src/components/Notification/Notification.style.ts

```diff
@@ -1,28 +1,33 @@
 import { css, SerializedStyles } from '@emotion/react';
```

Comment:
check styling is working as expected ->

![image](https://user-images.githubusercontent.com/6433679/134146708-18492296-51c6-4c05-b77b-fc3958a5c81f.png)